src/ray/common/scheduling/cluster_resource_scheduler.h

```diff
@@ -343,25 +356,46 @@ class ClusterResourceScheduler {
   ///
   /// \param available A list of available capacities for resource's instances.
   /// \param resource_instances List of the resource instances being updated.
-  void AddAvailableResourceInstances(std::vector<double> available,
-                                     ResourceInstanceCapacities *resource_instances);
+  ///
+  /// \return Overflow capacities of "resource_instances" after adding instance 
```

Comment:
I don't understand this comment. Can you explain what overflow capacity is? It seemed like it could be either that more resources than the total capacity were marked as available, or it's the new value for (capacity - available).

Reply:
done